tst/testinstall/varargs.tst

```diff
@@ -74,22 +74,10 @@ gap> function(a,b....) end;
 Syntax error: ) expected in stream:1
 function(a,b....) end;
                ^
-gap> f := function(a,b..) end;
-Syntax error: Three dots required for variadic argument list in stream:1
-f := function(a,b..) end;
-                 ^^
```

Comment:
I remove this test because it is tests the same thing as the test starting in line 57.